src/widgets/_lists.scss

```diff
@@ -0,0 +1,23 @@
+list {
+    background-color: bg-color(1);
+
+    popover & {
+        background-color: transparent;
+    }
+
+    row {
+        &:selected {
+            background-color: rgba($fg-color, 0.2);
```

Comment:
This feels quite a bit darker than the focus color. Is there a strong reason to not always use the same selected colors? It looks like Adwaita does at least.

Reply:
Otherwise, we could potentially try to desaturate the selected color here as well.

Reply:
Yes because you need to be able to tell the difference between keyboard focus and selection. This is especially apparent in views like Files' column view